### DISCUSSION TITLE
Does CefSharp have an undocumented dependency, I get an error System.IO.FileNotFoundException

### DISCUSSION THREAD

**Discussion by serkansubasi** [cefsharp/CefSharp#3907] [General]
While prebinding for settings I got an error

System.IO.FileNotFoundException: 'Could not load file or assembly 'CefSharp, Version=95.7.141.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The system cannot find the file specified.

I am not sure it is a bug or am I missing something?

Vs 2022
Winforms

FusionLog
=== Pre-bind state information === 
LOG: DisplayName = CefSharp, Version=95.7.141.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138 (Fully-specified) 
LOG: Appbase = file:///F:/SourceCodeRootPlaceHolder/bin/Debug/ 
LOG: Initial PrivatePath = NULL Calling assembly : xyxyx, Version=4.2.0.0, Culture=neutral, PublicKeyToken=null. === 
LOG: This bind starts in default load context. 
LOG: Using application configuration file: C:\SourceCodeRootPlaceHolder\bin\Debug\xyxyx.exe.Config 
LOG: Using host configuration file: 
LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework64\v4.0.30319\config\machine.config. 
LOG: Post-policy reference: CefSharp, Version=95.7.141.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138 
LOG: Attempting download of new URL file:///F:/SourceCodeRootPlaceHolder/bin/Debug/CefSharp.DLL. 
LOG: Attempting download of new URL file:///F:/SourceCodeRootPlaceHolder/bin/Debug/CefSharp/CefSharp.DLL. 
LOG: Attempting download of new URL file:///F:/SourceCodeRootPlaceHolder/bin/Debug/CefSharp.EXE. 
LOG: Attempting download of new URL file:///F:/SourceCodeRootPlaceHolder/bin/Debug/CefSharp/CefSharp.EXE.

**Comment by amaitland** [comment]
> Does CefSharp have an undocumented dependency

No. Read  https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29 in full. 

> While prebinding for settings I got an error

What is prebinding for settings exactly? 

> I am not sure it is a bug or am I missing something?

Unlikely a bug. 

See https://github.com/cefsharp/CefSharp.MinimalExample for a working example.

**Reply by serkansubasi** [reply]
Thank you Alex.
I found the cause.
Copy Local property of Cefsharp left out false, I did not know how it happened, I upgraded with nuget package manager, from version 63.0.3.0 to 95.7.141.0
